restapi/endpoints_test.go

```diff
@@ -482,6 +482,7 @@ func (suite *endpointTestSuite) TestGetVisibleStoriesWithTags() {
 	suite.db.Create(&jsonStories)
 	suite.db.Create(&jsonTags)
 	var response = suite.endpoint.GET("/stories").
+		WithQuery("visibility", true).
```

Comment:
Can we add a test response test for `WithQuery("visibility", false).`?

Reply:
Also shouldn't there be three modes in this enum? We can get visible stories, not visible stories or all stories. We should then have `visibility=true/false` for the first two conditions, and then if they don't pass in anything at all we can return all stories regardless of visibility

Reply:
yes we already have a test for invisible stories. Look at the func TestGetAllInvisibleStories().

Reply:
and thats exactly how it is. If there is no query param visibility, then we return all stories, otherwise it returns either visible or non-visible depending on whether visibility was passed in as true or false

Reply:
Ah ok I'm a dummy, that's my bad for just skimming and not actually reading lol.